### PR TITLE
♿️- Settings - Change email / Change password input fields

### PR DESCRIPTION
**Pull request by cdolm92**
# 📲 What

Accessibility improvements to textfields in Change Email and Change Password screens.

# 🤔 Why

Previously VoiceOver read both the title labels and textfields we found that this was not a good pattern because it adds to the complexity of focusable elements and doesn’t provide non-visual value.

# 🛠 How

We disabled accessibility on all title labels. We then set the accessibility label of the textfields to the title labels text.

# 👀 See
# ✅ Acceptance criteria
on all the following: iOS 10, iOS 11 & iOS 12
with Voice Over turned ON

**Test Change Email Screen**

Test Tap on row
- [x] Navigate to `Settings > Account > Change email`
- [x] Tap `New email` row, should read `New email - Email Address - Text Field - double tap to edit`
- [x] Tap `Current password` row, should read `Current password - Password - Text Field - double tap to edit`

Test Double Tap on TextField
- [x] Navigate to `Settings > Account > Change email`
- [x] Tap `New email` row, should read `New email - Email Address - Text Field - double tap to edit`
- [x] Double-tap should now read `New Email - text field - is editing - "Email Address" - character mode - insertion point at start`
- [x] Tap `Current Password - Secure Text Field - is editing - "Password" - character mode - insertion point at start`

Test Tap on row when text is present
- [x] Navigate to `Settings > Account > Change email`
- [x] Tap `New email` row, should read `New email - (email address read) - Text Field - double tap to edit`
- [x] Tap `Current password` row, should read `Current password - (number of characters) - Secure Text Field - double tap to edit`

Test Tap on textfield when text is present
- [x] Navigate to `Settings > Account > Change email`
- [x] Enter some text on each textfield
- [x] Double Tap on one of the textfields so that cursor is present
- [x] For **New email**  row  should read: `New Email - text field - is editing - (character) - character mode - insertion point at end`
- [x] For **Current password**  row should read:  `Current Password - Secure Text Field - is editing - (number of characters) - character mode - insertion point at end`

**Test Change Password Screen**

Test Tap on row
- [x] Navigate to `Settings > Account > Change password`
- [x] Tap `Current password` row, should read `Current password - Password - Text Field - double tap to edit`
- [x] Tap `New password` row, should read `New password - Password - Text Field - double tap to edit`
- [x] Tap `Confirm new password` row, should read `Confirm new password - Password - Text Field - double tap to edit`

Test Tap on row when text is present
- [x] Navigate to `Settings > Account > Change password`
- [x] Tap `Current password` row, should read `Current password - (number of characters - Secure Text Field - double tap to edit`
- [x] Tap `New password` row, should read `New password -(number of characters) character(s) - Secure Text Field - double tap to edit`
- [x] Tap `Confirm new password` row, should read `Confirm new password  -(number of characters) - Secure Text Field - double tap to edit `

- [x] The focus area has the same height for all fields `Current password`, `New password` & `Confirm new password`

Test Tap on textfield when text is present
- [x] Navigate to `Settings > Account > Change password`
- [x] Enter some text on each textfield
- [x] Double Tap on one of the textfields so that cursor is present
- [x] For **Current password**  row should read:  `Current Password - Secure Text Field - is editing - (number of characters) - character mode - insertion point at end`
- [x] For **New password**  row  should read: `New Password - Secure Text Field - is editing - (number of characters) - character mode - insertion point at end`
- [x] For **Confirm new password**  row  should read: `Confirm New Password - Secure Text Field - is editing - (number of characters) - character mode - insertion point at end`

### CODE REVIEW

**Comment by dusi** [comment]
Right now the acceptance criteria only accounts for the default state (placeholder).,.can we potentially update it with additional state (after user has entered some text)?

File: Kickstarter-iOS/Views/Cells/FindFriendsCell.swift [hunk 1]
```diff
@@ -13,8 +13,10 @@ final class FindFriendsCell: UITableViewCell, ValueCell, NibLoading {
   private let viewModel: FindFriendsCellViewModelType = FindFriendsCellViewModel()
 
   func configureWith(value: SettingsCellValue) {
-    let user = value.user
-    self.viewModel.inputs.configure(with: user)
+    _ = self
```
Comment (dusi) [inline Kickstarter-iOS/Views/Cells/FindFriendsCell.swift]: Are the following changes necessary?

* `FindFriendsCell`
* `SettingsAccountWarningCell`
* `SettingsTableViewCell`
* `SettingsViewController`
* some of the `Settings.storyboard` (mostly `UILabel`/`UITextField` prop names)
* `HelpType`
* `SettingsCellType`
* `README.md` (just the installation instrucitons) necessary?

They appear as duplicates from previous PRs so we should only include changes related to change email/password input fields a11y.

**Comment by dusi** [comment]
[♿️ Settings > Account > Change email/password - Input fields](https://trello.com/c/xuqzX5EB/1157-%E2%99%BF%EF%B8%8F-settings-account-change-email-password-input-fields)

**Comment by Scollaco** [comment]
Could we update the AC when the TextFields contain text? If we tap on a text field with text, the VoiceOver will say in case of Passwors text field:
`TextField name - is editing - x characters  - keyboard mode - insertion point localization`. For example: `Current Password - Secure Text Field - is editing - 6 characters - character mode - insertion point at end` 

And email textField:
`TextField name  - text field - is editing - input text - keyboard mode - insertion point localization`:
`New Email - text field - is editing - some text - character mode - insertion point at end`

**Pull request merged**